src/shared/vendor/third_party/webcomponents.js

```diff
@@ -6374,4 +6374,6 @@ CustomElements.addModule(function(scope) {
   window.Platform = scope;
 })(window.WebComponents);
 
-module.exports = window.WebComponents;
+if (typeof exports !== 'undefined') {
```

Comment:
Shouldn't this be `typeof module`?  Also, fixed in https://github.com/facebook/react/pull/4150